flang/test/Lower/OpenMP/auto-omp.f90

```diff
@@ -0,0 +1,52 @@
+! RUN: %flang_fc1 -O1 -mllvm --enable-affine-opt -emit-llvm -fopenmp -o - %s \
```

Comment:
We usually do not test end to end in this directory. If you really need to test the whole pipeline then you have to move this test to the Integration directory. Also, it might be good to check minimally (only the presence of OpenMP runtime calls) to avoid this test being a burden for others making unrelated changes.